ui/src/app/views/notebook-redirect/component.html

```diff
@@ -0,0 +1,15 @@
+<h2 *ngIf="notebookName">Loading Notebook '{{notebookName}}'</h2>
+<h2 *ngIf="!notebookName">Creating a New Notebook</h2>
+<div [ngSwitch]="progress" class="spinner-container">
+  <span class="spinner spinner-lg"></span>
+  <p *ngSwitchCase="Progress.Initializing" class="spinner-text">
```

Comment:
More question than comment (still learning the nitty gritty of angular) - if this is Progress.Unknown, will the spinner not show? Or will it never be that value because you set it to one of the other 3 in OnInit?

Reply:
It will be Unknown when we render initially since I only set the progress in a callback to `listClusters()`. These `ngSwitchCase` statements are on the spinner text, so I'll conditionally render one of those. The actual spinner itself is the `span` one line up.

So for progress == Unknown, we're showing a spinner here but no text underneath it.

Reply:
👍